src/lib/db/event-store.ts

```diff
@@ -149,6 +149,16 @@ class EventStore implements IEventStore {
         }
     }
 
+    async getMaxRevisionId(largerThan: number = 0): Promise<number> {
+        const row = await this.db(TABLE)
+            .max('id')
+            .whereNotNull('feature_name')
+            .orWhere('type', 'segment-update')
+            .andWhere('id', '>=', largerThan)
+            .first();
```

Comment:
Not sure if the ids are sorted desc or asc, I wonder if `max` is better.

Reply:
My understanding is the same, `max` would be similar in cost but might be more meaningful